Blockzilla/OverlayView.swift

```diff
@@ -183,50 +200,83 @@ class OverlayView: UIView {
                     self.copyButton.setAttributedTitle(attributedTitle, for: .normal)
                     showCopyButton = url.isWebPage()
                 }
+
+                let firstElementHidden = self.searchButtonGroup[0].isHidden
+                // Handle updating of other search buttons based on how many search suggestions there are.
+                let buttonsVisibleBefore = self.searchSuggestionsVisible
+                self.searchSuggestionsVisible = min(self.searchQueryArray.count, self.searchButtonGroup.count)
+                if self.searchQueryArray[0] == "" {self.searchSuggestionsVisible = 0}
+                // To flag whether FindInPage and Copy need to move if more/less search suggestions showing.
+                let numberSearchSuggestionsHasChanged = buttonsVisibleBefore != self.searchSuggestionsVisible
```

Comment:
Very small nit: consider renaming this ```numberOfSearchSuggetionsHasChanged``` 

best to make Swift read as much like English as possible 😄